frontend/device/remarkable/device.lua

```diff
@@ -22,20 +22,37 @@ local ABS_Y = 01
 local ABS_MT_POSITION_X = 53
 local ABS_MT_POSITION_Y = 54
 -- Resolutions from libremarkable src/framebuffer/common.rs
+local screen_width = 1404
```

Comment:
```suggestion
local screen_width = 1404 -- unscaled_size_check: ignore
```
For all four of them of course. ;-)

Reply:
Done.